metadata-ingestion/src/datahub/testing/mcp_diff.py

```diff
@@ -162,25 +163,31 @@ def create(
     ) -> "MCPDiff":
         ignore_paths = [cls.convert_path(path) for path in ignore_paths]
 
+        is_delta_valid = True
         aspect_changes: Dict[str, Dict[str, MCPAspectDiff]] = defaultdict(dict)
         for urn in golden.keys() | output.keys():
             golden_map = golden.get(urn, {})
             output_map = output.get(urn, {})
             for aspect_name in golden_map.keys() | output_map.keys():
+                t1 = golden_map.get(aspect_name, [])
+                t2 = output_map.get(aspect_name, [])
                 diff = DeepDiff(
-                    t1=golden_map.get(aspect_name, []),
-                    t2=output_map.get(aspect_name, []),
+                    t1=t1,
+                    t2=t2,
                     exclude_regex_paths=ignore_paths,
                     ignore_order=True,
                     custom_operators=[DeltaInfoOperator()],
                 )
                 if diff:
                     aspect_changes[urn][aspect_name] = MCPAspectDiff.create(diff)
+                if len(t1) > 1 and len(t2) > 1:
```

Comment:
should this be an `or`?

Reply:
I was thinking I could handle this but you're right, I don't. Will make this change. Seems like I also have to regenerate goldens?